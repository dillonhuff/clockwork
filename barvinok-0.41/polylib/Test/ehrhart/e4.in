# 0 <= i <= 5
# 0 <= j <= N
#------------
4 5
1  1  0 0 0
1 -1  0 0 5
1  0  1 0 0
1  0 -1 1 0

1 3
1 1 0 

N
4th example
