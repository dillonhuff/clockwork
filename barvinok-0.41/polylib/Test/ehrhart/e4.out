---------------------------------------
Domain:
         N  >= 0
          1 >= 0

Vertices:
[ 5,  N ]
[ 5,  0 ]
[ 0,  N ]
[ 0,  0 ]

Ehrhart Polynomial:
( 6 * N + 6 )

